internal/kafka/internal/config/kafka.go

```diff
@@ -56,6 +61,8 @@ func (c *KafkaConfig) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&c.Quota.AllowDeveloperInstance, "allow-developer-instance", c.Quota.AllowDeveloperInstance, "Allow the creation of kafka developer instances")
 	fs.StringVar(&c.SupportedInstanceTypes.ConfigurationFile, "supported-kafka-instance-types-config-file", c.SupportedInstanceTypes.ConfigurationFile, "File containing the supported instance types configuration")
 	fs.StringVar(&c.BrowserUrl, "browser-url", c.BrowserUrl, "Browser url to kafka admin UI")
+	fs.BoolVar(&c.EnableKafkaOwnerConfig, "enable-kafka-owner-config", c.EnableKafkaOwnerConfig, "Enable configuration for setting kafka owners")
+	fs.StringVar(&c.KafkaOwnerListFile, "kafka-owner-list-file", c.KafkaOwnerListFile, "File containing list of kafka owners")
```

Comment:
great.
We may also need to:
- add the corresponding parameters and configmap of the file in the `service-template.yaml` file: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/3debe34479689bf45ee61292fe627537742fee82/templates/service-template.yml. 
- add the way  to override the parameters in the `make deploy/service` target in the `Makefile`. https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/3debe34479689bf45ee61292fe627537742fee82/Makefile#L714-L823
- add a documentation in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/3debe34479689bf45ee61292fe627537742fee82/docs/deploying-kas-fleet-manager-to-openshift.md#deploy-kas-fleet-manager